src/internal/clusterstate/v2.4.0.go

```diff
@@ -3,11 +3,14 @@ package clusterstate
 import (
 	"context"
 
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+	auth "github.com/pachyderm/pachyderm/v2/src/server/auth/server"
+
```

Comment:
Why is there an newline here?

Reply:
To separate the standard library from our libraries.